app/routers/file_uploader.py

```
@@ -0,0 +1,47 @@
+import traceback
+from fastapi import APIRouter, File, UploadFile, HTTPException
+from google.cloud import storage
+import os
+
+router = APIRouter()
+
+BUCKET_NAME = "mou-pdf"
+MAX_FILE_SIZE = 10 * 1024 * 1024  # 10MB
+
+@router.post("/upload/")
+async def upload_document(file: UploadFile = File(...)):
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)